lib/inquisition/active_record_doctor/runner.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails'
+require 'active_record'
+require 'active_record_doctor/tasks'
+require 'active_record_doctor/tasks/extraneous_indexes'
+require 'active_record_doctor/tasks/missing_foreign_keys'
+require 'active_record_doctor/tasks/unindexed_deleted_at'
+require 'active_record_doctor/tasks/unindexed_foreign_keys'
+
+module Inquisition
+  module ActiveRecordDoctor
+    class Runner < ::Inquisition::Runner
+      def initialize
+        ActiveRecord::Base.establish_connection(load_db_config)
+      end
+
+      def call
+        issues = []
+        ::ActiveRecordDoctor::Tasks.all.each do |ard_module|
+          ard_module.run.first.each do |table, column|
+            issues << Issue.new(level: Issue::LEVELS[:low], file: nil, line: nil, runner: self,
+                                message: create_message(ard_module, table, column))
+          end
+        end
+        issues
+      end
+
+      private
+
+      def create_message(ard_module, table, column)
+        issue_text = ard_module.to_s.split('::').last.split(/(?=[A-Z])/).map(&:downcase).join(' ')
+        "Table #{table} has #{issue_text} in column(s) #{column.join(', ')}"
```

Comment:
Does this message fit all kinds of problems? (like example https://github.com/gregnavis/active_record_doctor#detecting-models-referencing-undefined-tables)